src/components/Statistics/StatisticCardAll.tsx

```diff
@@ -0,0 +1,59 @@
+import { Card,CardContent, Grid, Typography } from '@material-ui/core'
+import React from 'react'
+import useStyles from './style'
+import {StatisticsCard} from './StatisticsCard'
+
+
+
+
+interface Props {
+    
+}
+
+export const StatisticCardAll = (props: Props) => {
```

Comment:
Please use `React.FC` as type of component

Reply:
Done